src/App.js

```diff
@@ -0,0 +1,13 @@
+import Countries from './components/Countries';
+import Footer from './components/Footer';
+
+function App() {
+  return (
+    <div className="App">
+      <Countries />
+      <Footer />
+    </div>
+  );
```

Comment:
- [ ] Please fix your mobile design to match the orginal one as required 

- Orginial design 👇 

![image](https://user-images.githubusercontent.com/75126481/235229283-0d0acd96-c5be-4583-a600-d8e6b28749fb.png)

- Your design 👇 

![image](https://user-images.githubusercontent.com/75126481/235230530-b102c408-d0a2-4cc5-8eff-ab486474664f.png)

![image](https://user-images.githubusercontent.com/75126481/235230026-48b4a257-ec0d-48aa-9e90-5d85212be587.png)

Reply:
- [ ] Good job working on most of the requirements requested by the previous reviewer. Unfortunately the country details page I think needs a right margin. As you can see, the Incident Rate cell aligns with the container. Kindly give some margin to make your design similar. See screenshot below.
![Screen Shot 2023-04-28 at 9 20 59 PM](https://user-images.githubusercontent.com/62849872/235283221-b775b804-5e06-4dba-8ef4-249475b54a38.png)